cmd/configure/configure.go

```diff
@@ -140,6 +141,10 @@ func argValidation(cmd *cobra.Command, args []string) error {
 		if regex.MatchString(arg) {
 			return fmt.Errorf("'%s' is not a valid argument. Did you mean 'signin'?", arg)
 		}
+		// If arg is missing protocol prefix, temporarily add one to validate the url
+		if !strings.HasPrefix(arg, "http") {
```

Comment:
Maybe better to check for http:// or https://? I don't know if there is any website out there that starts with http but technically you could have a website like `httpmycontroller.example.com`.